app/components/Dashboard.jsx

```diff
@@ -19,11 +18,13 @@ const mapStateToProps = (state) => ({
   consumerPendingDuration: state.consumer.pendingDuration,
 });
 
+// Mapping dispatch actions to trigger state changes for Producer and Consumer metrics
 const mapDispatchToProps = (dispatch) => ({
   addProducerData: (newData) => dispatch(actions.addProducerData(newData)),
   addConsumerData: (newData) => dispatch(actions.addConsumerData(newData)),
 });
 
+// Renders ProducerMetrics and ConsumerMetrics and updates props
```

Comment:
-add the word "components" after ConsumerMetrics for clarity
-what do you mean "updates props"? could you be a bit more precise here?